s4e-web/src/app/views/map-view/sentinel-search/search-results/search-results.component.html

```diff
@@ -86,9 +86,11 @@
                   (click)="showDetails.emit(result)"
                 ></button>
                 <a
-                  [attr.href]="result.url"
+                  [attr.href]="result.url ? result.url : 'javascript:undefined'"
                   (click)="isUserLoggedIn || interceptDownload($event)"
-                  target="_blank"
+                  [attr.target]="result.url ? '_blank' : ''"
+                  [attr.title]="result.url ? '' : 'Ten produkt nie posiada pliku'"
```

Comment:
Rather "Ta scena nie posiada artefaktu zip", as the scene will contain at least a single file anyway (metadata file).
Also, in the details modal we use the "artefakt" nomenclature and I would stick to it.